code/modules/paperwork/paper.dm

```
@@ -65,7 +65,7 @@
 		send_asset_list(user.client, S.assets)
 
 	var/data
-	if(!(user.say_understands(null, all_languages["Galactic Common"]) && !forceshow) || forcestars) //assuming all paper is written in common is better than hardcoded type checks
+	if((!user.say_understands(null, all_languages["Galactic Common"]) && !forceshow) || forcestars) //assuming all paper is written in common is better than hardcoded type checks
```

It had unbalanced parenthesis though, that should at least throw an error.